tests/topics/test_topic_list_endpoint.py

```diff
@@ -0,0 +1,236 @@
+"""Testing TopicList endpoint."""
+
+from datetime import datetime
+
+import jwt
+
+from src.config import Config
+from src.topics.models import Topic
+
+
+def test_get_topic_list_endpoint_with_no_args_returns_whole_list(client, db_with_three_users_and_three_topics):
+    authorization_token = jwt.encode({"user_id": 1}, Config.SECRET_KEY, algorithm="HS256")
+
+    result = client.get("api/topics", headers={"Authorization": f"Bearer {authorization_token}"})
+
+    assert result.status_code == 200
+    assert result.json == [
+        {
+            "id": 1,
+            "author_id": 1,
+            "created_at": "2023-05-06 00:12:38.078953",
+            "description": "test desc",
+            "title": "test title",
+        },
+        {
+            "id": 2,
+            "author_id": 2,
+            "created_at": "2023-05-06 00:12:38.078953",
+            "description": "test desc2",
+            "title": "test title2",
+        },
+        {
+            "id": 3,
+            "author_id": 3,
+            "created_at": "2023-05-07 00:12:38.078953",
+            "description": "test desc3",
+            "title": "test title3",
+        },
+    ]
+
+
+def test_get_topic_list_endpoint_with_order_by_returns_200(client, db_with_three_users_and_three_topics):
```

Comment:
Since we check not only the returned http code but the response body as well, we need to add `_with_correct_body` in the title.

Please also check other tests for this change